CHANGELOG.md

```diff
@@ -119,6 +119,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking Changes
 
+* (x/auth) [#13850](https://github.com/cosmos/cosmos-sdk/pull/13850/) Remove `MarshalYAML` methods from `x/auth` types.
```

Comment:
it was removed `MarshalYAML` from all modules, right?

Reply:
That's true. I will update that :)